i18n/ja/docusaurus-plugin-content-docs-plugin-guide/current/recipes/screen-capture-and-print.md

```diff
@@ -1,15 +1,16 @@
 ---
-title: Screen capture and print
+title: スクリーンキャプチャと印刷
 sidebar_position: 10
 ---
 
 # CaptureScreen API
 
-Plugins can get the image of the canvas with this API.
+プラグインは、このAPIを使用してキャンバスの画像を取得することができます。
 
 :::note
 
-The output image comes from the canvas so anything implemented out of canvas (eg. infobox) can not be included. If you want to take a screenshot of all HTML elements you may try implement with some third-party libraries in plugin code.
+画像はキャンバスから吐き出されるため、キャンバス外で実装されたもの(例：インフォボックス)は含まれません。
+すべてのHTML要素のスクリーンショットを取りたい場合は、プラグインコードでいくつかのサードパーティライブラリを実装してみてください。

```

Comment:
```suggestion

```

It is not supposed to take screenshots of all elements for now. Fix the original text also.